sda-commons-server-kms-aws/src/test/java/org/sdase/commons/server/kms/aws/KmsAwsPlainStringAndDecryptTest.java

```diff
@@ -0,0 +1,134 @@
+package org.sdase.commons.server.kms.aws;
+
+import java.util.Collections;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class KmsAwsPlainStringAndDecryptTest {
+
+  @Test
+  public void returnIfCryptoIsDisabledEncryption() {
+    String encryptionInput = "encryptionInput";
+
+    KmsAwsEncryptionService encryptionService =
+        KmsAwsEncryptionService.builder().withEncryptionDisabled().build();
+
+    byte[] encrypted = encryptionService.encrypt(encryptionInput.getBytes(), null);
+    Assert.assertEquals(encryptionInput, new String(encrypted));
+  }
+
+  @Test
+  public void returnIfCryptoIsDisabledDecryption() throws InvalidEncryptionContextException {
+    String decryptionInput = "decryptionInput";
+    KmsAwsDecryptionService decryptionService =
+        KmsAwsDecryptionService.builder().withDecryptionDisabled().build();
+
+    byte[] decrypted = decryptionService.decrypt(decryptionInput.getBytes(), null);
+    Assert.assertEquals(decryptionInput, new String(decrypted));
+  }
+
+  @Test
+  public void encryptAndDecryptWithCachingTest() throws InvalidEncryptionContextException {
+    String encryptionInput = "encryptionInput";
+    KmsAwsEncryptionServiceMock kmsAwsEncryptionServiceMock = new KmsAwsEncryptionServiceMock(true);
+    KmsAwsEncryptionService kmsAwsEncryptionService =
+        kmsAwsEncryptionServiceMock.getKmsAwsEncryptionService();
+
+    Map<String, String> context = Collections.singletonMap("key", "value");
+
+    byte[] encrypt = kmsAwsEncryptionService.encrypt(encryptionInput.getBytes(), context);
+
+    KmsAwsDecryptionService kmsAwsDecryptionService =
+        new KmsAwsDecryptionServiceMock(true, kmsAwsEncryptionServiceMock.getMockKmsClient())
+            .getKmsAwsDecryptionService();
+
+    byte[] decrypt = kmsAwsDecryptionService.decrypt(encrypt, context);
+
+    Assert.assertEquals(encryptionInput, new String(decrypt));
+  }
+
+  @Test
+  public void encryptAndDecryptWithoutCachingTest() throws InvalidEncryptionContextException {
+    String encryptionInput = "encryptionInput";
+    KmsAwsEncryptionServiceMock kmsAwsEncryptionServiceMock =
+        new KmsAwsEncryptionServiceMock(false);
+    KmsAwsEncryptionService kmsAwsEncryptionService =
+        kmsAwsEncryptionServiceMock.getKmsAwsEncryptionService();
+
+    Map<String, String> context = Collections.singletonMap("key", "value");
+
+    byte[] encrypt = kmsAwsEncryptionService.encrypt(encryptionInput.getBytes(), context);
+
+    KmsAwsDecryptionService kmsAwsDecryptionService =
+        new KmsAwsDecryptionServiceMock(false, kmsAwsEncryptionServiceMock.getMockKmsClient())
+            .getKmsAwsDecryptionService();
+
+    byte[] decrypt = kmsAwsDecryptionService.decrypt(encrypt, context);
+
+    Assert.assertEquals(encryptionInput, new String(decrypt));
```

Comment:
We normally use [assertJ](https://joel-costigliola.github.io/assertj/) for assertions